tests/fixtures/bugs.json

```diff
@@ -51,4 +51,6 @@
f_qa_whiteboard": "", "id": 1242711, "is_creator_accessible": true, "severity": "S3", "creator_detail": {"nick": "yzen", "email": "yura.zenevich@gmail.com", "id": 458165, "real_name": "Yura Zenevich [:yzen] (please use \"needinfo\")", "name": "yura.zenevich@gmail.com"}, "cf_crash_signature": "", "see_also": [], "depends_on": [], "cf_fx_points": "---", "cf_cab_review": "---", "cf_status_firefox122": "---", "target_milestone": "---", "op_sys": "Unspecified", "cf_performance_impact": "---", "cf_user_story": "", "component": "Inspector", "cf_status_firefox120": "---", "url": "", "priority": "P3", "cf_tracking_firefox120": "---", "cf_accessibility_severity": "s4", "cf_has_str": "---", "cf_tracking_firefox_relnote": "---", "groups": [], "status": "NEW", "history": [{"who": "advck1123@gmail.com", "changes": [{"removed": "", "added": "advck1123@gmail.com", "field_name": "cc"}], "when": "2016-01-27T02:17:02Z"}, {"when": "2016-04-08T19:51:35Z", "changes": [{"field_name": "priority", "added": "P3", "removed": "--"}, {"field_name": "cc", "added": "pbrosset@mozilla.com", "removed": ""}, {"added": "[btpp-backlog]", "removed": "", "field_name": "whiteboard"}], "who": "patrickbrosset+bugzilla@gmail.com"}, {"when": "2018-06-13T17:27:05Z", "changes": [{"field_name": "product", "added": "DevTools", "removed": "Firefox"}], "who": "automation@bmo.tld"}, {"when": "2020-07-14T01:17:19Z", "changes": [{"field_name": "whiteboard", "removed": "[btpp-backlog]", "added": "[btpp-backlog][a11y_2020]"}], "who": "yura.zenevich@gmail.com"}, {"when": "2022-03-28T00:38:05Z", "who": "jteh@mozilla.com", "changes": [{"added": "jteh@mozilla.com", "removed": "", "field_name": "cc"}, {"field_name": "whiteboard", "added": "[btpp-backlog][a11y_2020] [access-s4]", "removed": "[btpp-backlog][a11y_2020]"}]}, {"who": "automation@bmo.tld", "changes": [{"removed": "normal", "added": "S3", "field_name": "severity"}], "when": "2022-10-11T17:58:28Z"}, {"when": "2023-06-02T14:34:37Z", "who": "automation@bmo.tld", "changes": [{"added": "[btpp-backlog][a11y_2020] ", "removed": "[btpp-backlog][a11y_2020] [access-s4]", "field_name": "whiteboard"}]}, {"changes": [{"field_name": "cc", "removed": "", "added": "automation@bmo.tld"}, {"field_name": "cf_accessibility_severity", "removed": "---", "added": "s4"}], "who": "automation@bmo.tld", "when": "2023-06-02T14:43:16Z"}], "comments": [{"creation_time": "2016-01-25T21:11:41Z", "text": "Right in the inspector tab (and likely others) there are unnecessary tabs that need to be performed to move between frames and tabpanels. We need to figure out what they are and hide them from screen reader (with presentation role) to avoid ever tabbing to them.", "count": 0, "id": 11105922}, {"id": 11311941, "count": 1, "text": "Triaging (filter on CLIMBING SHOES).", "creation_time": "2016-04-08T19:51:35Z"}, {"text": "Most recent extra tab stops:\n* extra tab stop on DOM tree container", "creation_time": "2020-07-14T01:17:01Z", "id": 14935678, "count": 2}], "attachments": []}
```

Comment:
Can we reuse existing bugs instead of adding new ones?

Reply:
Last I checked, there weren't any accessibility bugs. I only added two accessibility bugs and reused the existing ones for non-accessibility bugs.